packages/datadog-plugin-cypress/test/app/cypress/plugins/index.js

```diff
@@ -1,7 +1,16 @@
+/* eslint-disable */
+
 module.exports = (on, config) => {
   // We can't use the tracer available in the testing process, because this code is
   // run in a different process. We need to init a different tracer reporting to the
   // url set by the plugin agent
-  require('../../../../../dd-trace').init({ startupLogs: false })
+
+  const tracer = require('../../../../../dd-trace').init({
+    startupLogs: false,
+    isCiVisibility: true,
+  })
+  tracer.use('fs', false)
+  tracer.use('child_process', false)
```

Comment:
this is just a testing quirk. We want the tracer init in the plugin test to look like `ci/init` as much as possible